packages/fx-core/src/plugins/resource/frontend/plugin.ts

```diff
@@ -229,6 +233,57 @@ export class FrontendPluginImpl {
     const progressHandler = await ProgressHelper.createPreDeployProgressHandler(ctx);
 
     const config = await FrontendConfig.fromPluginContext(ctx);
+
+    let functionEnv: FunctionEnvironment | undefined;
+    let aadEnv: AADEnvironment | undefined;
+    let runtimeEnv: RuntimeEnvironment | undefined;
+
+    const functionPlugin = ctx.configOfOtherPlugins.get(DependentPluginInfo.FunctionPluginName);
+    if (functionPlugin) {
+      functionEnv = {
+        defaultName: functionPlugin.get(DependentPluginInfo.FunctionDefaultName) as string,
+        endpoint: functionPlugin.get(DependentPluginInfo.FunctionEndpoint) as string,
+      };
+    }
+
+    if (isArmSupportEnabled()) {
+      const endpoint = getArmOutput(ctx, ArmOutput.SimpleAuthEndpoint) as string;
```

Comment:
Please confirm with Chaoyi or Di ,whether ArmOutput could be used here.

Reply:
Sure

Reply:
Please use `ctx.config.getString(xxx)` to get config.
`getArmOutput()` should only be used during provision. After provision, the arm output will be deleted and cannot be accessed. @xzf0587 is working on making `getArmOutput()` only available during provision on code level.

Reply:
OK, thanks!

Reply:
Updated